components/modal/Modal.tsx

```diff
@@ -70,6 +70,7 @@ export interface ModalFuncProps {
   style?: React.CSSProperties;
   type?: string;
   keyboard?: boolean;
+  autoFocusOk?: boolean;
```

Comment:
Could you add it to documentation?

Reply:
Done, feel free to update the description.